client/src/components/Calendar/index.js

```diff
@@ -0,0 +1,184 @@
+import React, { Component } from 'react';
+import {Icon, TableHeaderCell, Table, Header, Segment } from 'semantic-ui-react';
+import styled from 'styled-components';
+import Moment from 'moment';
+import {extendMoment} from 'moment-range';
+const moment = extendMoment(Moment);
+
+const HeaderWrapper = styled.div`
+    height: 80px;
+    width: 95%;
+    margin: 3%;
+    overflow: hidden;
+`;
+
+const HeaderArrow = {
+    color: "#FA8072", 
+    paddingTop: "5%"
+};
+
+const Title = styled.span`
+    position: absolute;
+    font-family: Helvetica;
+    font-size: 3em;
+    color: #FA8072;
+    text-align: center;
+    margin-left: 20%;
+    margin-top: 5%;
+    font-weight: bold;
+    text-transform: uppercase;
+`;
+
+class Calendar extends Component {
+    constructor(props){
+        super(props);
+        this.state={
+            month: moment().month(),
+            year: moment().year(),        
+        }
+    }
+
+    previousMonth = ()=>{
+        if(this.state.month === 0){
+            this.setState({
+                month: 11,
+                year: this.state.year-1
+            });
+        } else {
+            this.setState({
+                month: this.state.month - 1,
+                year: this.state.year
+            });
+        }
+        console.log('this.state.month: ' + this.state.month);
+    }
+    nextMonth = ()=>{
+        let month, year;
+        if (this.state.month === 11){
+            this.setState({
+                month: 0,
+                year: this.state.year + 1
+            });
+        }
+        else{
+            this.setState({
+                month: this.state.month + 1,
+                year: this.state.year
+            });
+        }
+    }
+
+    render(){
+        let daysOfWeek = ["Sun", "Mon", "Tues", "Wed", "Thurs", "Fri", "Sat"],
```

Comment:
@amtadros If you have some time later today I would love to talk through the process to generate the calendar and maybe talk about abstracting some of it out into its own file to make the render method a little cleaner here.

Reply:
A good start would be for daysOfWeek to be pulled out into a constants file; as it is, it's recreated every time the component renders.